splink/comparison_template_library.py

```diff
@@ -0,0 +1,173 @@
+# Script defining comparison templates that act as wrapper functions which produce
+# comparisons based on the type of data in a column with default values to make
+# it simpler to use splink out-of-the-box
+
+from __future__ import annotations
+
+from .comparison import Comparison  # change to self
+from .comparison_library_utils import datediff_error_logger
+from .misc import ensure_is_iterable
+
+
+class DateComparisonBase(Comparison):
+    def __init__(
+        self,
+        col_name,
+        include_exact_match_level=True,
+        term_frequency_adjustments=False,
+        separate_1st_january=True,
+        levenshtein_thresholds=[2],
```

Comment:
Have you checked if it works if `len(levenshtein_thresholds) != len(m_probability_or_probabilities_lev)`